plugins/religion.py

```diff
@@ -36,3 +36,381 @@ def koran(inp):  # Koran look-up plugin by Ghetto Wizard
+People are rebellious. When rulers are active People are rebellious. 
+People ignore death. When searching only for life's bounty People ignore death. Only those who do not strive after life Truly respect life.""",
+"""Humans are born soft and weak. They die stiff and strong. 
+The ten thousand plants and trees Are born soft tender, And die withered and sere. 
+The stiff and strong Are Death's companions The soft and weak Are Life's companions. 
+Therefore the strongest armies do not conquer, The greatest trees are cut down. 
+The strong and great sink down. The soft and weak rise up.""",
+"""Heaven's Tao Is a stretched bow, Pulling down on the top Pulling up on the bottom. 
+If it's too much, cut. If it's not enough, Add on to it: Heaven's Tao. 
+The Human Route Is not like this, Depriving the poor, Offering to the rich. Who has a surplus And still offers it to the world? Only those with Tao. 
+Therefore the Sage Acts and expects nothing, Accomplishes and does not linger, Has no desire to seem worthy.""",
+"""Nothing in the world is soft and weak as water. But when attacking the hard and strong Nothing can conquer so easily. 
+Weak overcomes strong, soft overcomes hard. Everyone knows this, no one attains it. 
+Therefore the Sage says: Accept a country's filth And become master of its sacred soil. Accepts country's ill fortune And become king under heaven. True words resemble their opposites.""",
+"""Appears great hatred sand hatred will remain. How can this be good? 
+Therefore the Sage Holds the tally But does not judge people. 
+Those who have Te Control the tally. Those who lack Te Collect their due. 
+Heaven has no favourites But endures in good people.""",
+"""Small country, few people - Hundreds of devices, But none are used. People ponder on death And don't travel far. 
+They have carriages and boats, But no one goes on board; Weapons and armour, But no one brandishes them. 
+They use knotted cords for counting. Sweet is their food, Beautiful their clothes, Peaceful their homes, Delightful their customs. 
+Neighboring countries are so close You can hear their chickens and dogs. But people grow old and die Without needing to come and go.""",
+"""Sincere words and not pretty. Pretty words are not sincere. 
+Good people do not quarrel. Quarrelsome people are not good. 
+The wise are not learned. The learned are not wise. 
+The Sage is not acquisitive - Has enough By doing for others, Has even more By giving to others. 
+Heaven's Tao Benefits and does not harm. The Sage's Tao Acts and does not contend."""]
+
+@hook.command('laozi')
+@hook.command
+def tao(inp):
+    ".tao <chapter:line> -- gets line <line> from chapter <chapter> of the Tao Te Ching (Addiss)"
+
+    resp = ""
+
+    if ':' not in inp:
+        return tao.__doc__
+
+    chapter, line = inp.split(':', 1)
+
+    try:
+        chapter = int(chapter)
+        line = int(line)
+    except ValueError:
+        return tao.__doc__
+
+    if chapter <= 81:
```

Comment:
Move the chapter check above and fail early. Instead of using hardcoded numbers, use the length of the list:

```python
if chapter > len(TAO):
    return "Invalid chapter {}, chapter numbers are between 1 and {}".format(chapter, len(TAO))
```